spec/openapi.yaml

```diff
@@ -148,6 +148,36 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /postedtime:
+    get:
+      operationId: fetch-posted-time
+      summary: Returns posted time for the team associated with the provided api key.
+      description: This is a long polling call. The connection will be held open until there is a new posted time event. This is an alternative for the webhook mechanism.
+      tags:
+        - Posted Time
+      parameters:
+        - in: query
+          name: limit
+          schema:
+            type: integer
+          required: false
+          description: Maximum amount of posted time entries to retrieve. If not set or 0 all available will be returned.
```

Comment:
> If not set or 0 all available will be returned.

Can we instead say:

> If not set, the API will return up to a maximum of 100 entries for each request.

If limit of 0 is received with the request we should respond with a 400 bad request.

Reply:
I would set max at 25 initially, as the results returned with be locked for some time and then released.  Ideally the lock should not cover a huge number.  Client can always use concurrent pull calls if they want faster response times.